CMPUT301F24bubble/bubbletracks
# Notifications

Closes #8  and #9.

Looks at #32, #33, #34, and #25. The exception is that organizers may be able to send custom notification messages.

- New classes: Notification, NotificationDB, and OrganizerNotificationActivity. Additional changes to existing classes include MainActivity, OrganizerEditActivity, and OrganizerEntrantListActivity

To test:
- You need two devices: organizer and entrant(s). Create an event for the organizer and add the entrants to the event. 

- go to organizer hosting list and click on the notification button --> you will be directed to the notifications screen. you can test waitlist entrants by checking off "Notify Entrants on the Waiting List" and then the "Notify Attendees" button. You should see the notification on your entrant device.

- For simplicity, you can add the entrant to the specific entrant list to your organizer event document in Firestore (wait, invited, enrolled, rejected) and then send the different types of notifications.
- For a real runthrough, draw entrant(s) in drawing entrants activity, then go back to notifications actvity to notify the specific entrants. You should also see the notification on your entrant device.
